dags/state_table_dag.py

```
@@ -90,240 +102,76 @@
-    partition=True,
-    cluster=True,
-    dataset_type="pub",
-)
-send_config_settings_to_pub_task = build_gcs_to_bq_task(
-    dag,
-    changes_task.task_id,
-    public_project,
-    public_dataset,
-    table_names["config_settings"],
-    "/*-config_settings.txt",
-    partition=True,
-    cluster=True,
-    dataset_type="pub",
-)
-send_ttl_to_pub_task = build_gcs_to_bq_task(
-    dag,
-    changes_task.task_id,
-    public_project,
-    public_dataset,
-    table_names["ttl"],
-    "/*-ttl.txt",
-    partition=True,
-    cluster=True,
-    dataset_type="pub",
-)
-
-(
-    date_task
-    >> changes_task
-    >> write_acc_stats
-    >> delete_acc_pub_task
-    >> send_acc_to_pub_task
-)
-
-
-(
-    date_task
-    >> changes_task
-    >> write_bal_stats
-    >> delete_bal_pub_task
-    >> send_bal_to_pub_task
-)
-
-
-(
-    date_task
-    >> changes_task
-    >> write_off_stats
-    >> delete_off_pub_task
-    >> send_off_to_pub_task
-)
-
-
-(
-    date_task
-    >> changes_task
-    >> write_pool_stats
-    >> delete_pool_pub_task
-    >> send_pool_to_pub_task
-)
-
-
-(
-    date_task
-    >> changes_task
-    >> write_sign_stats
-    >> delete_sign_pub_task
-    >> send_sign_to_pub_task
-)
-
-
-(
-    date_task
-    >> changes_task
-    >> write_trust_stats
-    >> delete_trust_pub_task
-    >> send_trust_to_pub_task
-)
+del_ins_tasks = {}
+
+# Define the suffixes for the DAG related tables
+source_object_suffix_mapping = {
+    "accounts": "/*-accounts.txt",
+    "claimable_balances": "/*-claimable_balances.txt",
+    "offers": "/*-offers.txt",
+    "liquidity_pools": "/*-liquidity_pools.txt",
+    "signers": "/*-signers.txt",
+    "trustlines": "/*-trustlines.txt",
+    "contract_data": "/*-contract_data.txt",
+    "contract_code": "/*-contract_code.txt",
+    "config_settings": "/*-config_settings.txt",
+    "ttl": "/*-ttl.txt",
+}
+
+# filter for only the required tables pertaining to the DAG
+table_id_and_suffixes = {
+    key: suffix
```

nit: I don't think this needs to create a new dict. `source_object_suffix_mapping` and `table_id_and_suffixes` are effectively the same thing I think.

Also this kind of fails silently if the `key` is not in `table_names` whereas I think right now if `"accounts"` wasn't defined in the airflow var for `table_names` it would give us an error

---

I will remove this and add a check if the keys are missing in the table_names variable.